client/src/components/manageProjects/selectProject.js

```diff
@@ -27,28 +32,53 @@ const SelectProject = ({ projects, accessLevel, user }) => {
 
   return (
     <div className="container--ManageProjects">
-      <h3>Manage Projects</h3>
-      <div className="project-sub-heading" style={{ margin: '0 auto' }}>
+      <h3>Project Management</h3>
+      <div className="project-sub-heading">
         {accessLevel === 'admin' ? (
-          <Link to="useradmin">
-            {' '}
+          // <Link to="useradmin">
+          <>
+            <button
+              className={`button ${toggle && 'active'}`}
+              style={{
+                fontSize: 'small',
+                width: 'auto',
+              }}
+              value='findProject'
+              onClick={handleToggle}
+            >
+              Find Project
+            </button>
+            {/* <Link to="useradmin"> */}
+
             <button
```

Comment:
Somehow this button is broken. crashed the whole app
<img width="803" alt="Screen Shot 2023-02-13 at 4 40 58 PM" src="https://user-images.githubusercontent.com/53061723/218608497-a7a082f7-aa91-42ee-beca-817f377d4192.png">